docker-test-env/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.9"
+services:
+  ic:
+    build: dfx-docker
+    ports:
+      - "8000:8000"
+    volumes:
+      - "./dfx-docker/canisters:/canisters:ro"
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://localhost:8000?canisterId=rrkah-fqaaa-aaaaa-aaaaq-cai" ]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+    extra_hosts:
+      - "rrkah-fqaaa-aaaaa-aaaaq-cai.localhost:127.0.0.1"
+      - "ryjl3-tyaaa-aaaaa-aaaba-cai.localhost:127.0.0.1"
```

Comment:
where do these values come from?

Reply:
The test driver is still running locally. Everything else is within docker compose:
so test driver (locally) -> selenium container ->  nginx container -> dfx container. I have added a readme explaining this in a later PR.

Reply:
The values are hardcoded to just the canisterIds that `dfx` happens to assign those canisters (deterministically).